backend/src/services/auth/auth.service.ts

```
@@ -1107,10 +1107,9 @@ export class AuthService {
   }
 
   /**
-   * Sign in with ID token from native SDK (Google One Tap, etc.)
-   * Limited to Google for now to unblock customer ask, can extend to other providers later as needed.
+   * Sign in with ID token from native SDK (Google One Tap, Apple Sign In, etc.)
    */
-  async signInWithIdToken(provider: 'google', idToken: string): Promise<CreateSessionResponse> {
+  async signInWithIdToken(provider: 'google' | 'apple', idToken: string): Promise<CreateSessionResponse> {
```

ESLint

[error] Replace `provider:·'google'·|·'apple',·idToken:·string` with `⏎····provider:·'google'·|·'apple',⏎····idToken:·string⏎··`

(prettier/prettier)